modules/gapi/perf/cpu/gapi_imgproc_perf_tests_cpu.cpp

```diff
@@ -134,6 +134,31 @@ INSTANTIATE_TEST_CASE_P(CannyPerfTestCPU, CannyPerfTest,
         Values(true, false),
         Values(cv::compile_args(IMGPROC_CPU))));
 
+INSTANTIATE_TEST_CASE_P(OptFlowLKPerfTestCPU, OptFlowLKPerfTest,
```

Comment:
All stuff you've added here should be moved to the separate file which is dedicated to test G-API operations for OpenCV *video* module on CPU : modules/gapi/perf/cpu/gapi_video_perf_tests_cpu.cpp (which you should create).